cli/src/generate/templates/binding.cc

```diff
@@ -1,28 +1,18 @@
 #include "tree_sitter/parser.h"
-#include <node.h>
-#include "nan.h"
+#include <napi.h>
 
-using namespace v8;
+using namespace Napi;
```

Comment:
I think you should remove this and just use `Napi::` everywhere

Reply:
Can be done. The examples often do `use namespace Napi`, but you obviously don't have too.